docs/docs/cmd/login.md

```diff
@@ -52,10 +52,15 @@ When logging in to Microsoft 365 using the user name and password, next to the a
 
 When logging in to Microsoft 365 using a certificate, the CLI for Microsoft 365 will store the contents of the certificate so that it can automatically re-authenticate if necessary. The contents of the certificate are removed by re-authenticating using the device code or by calling the [logout](logout.md) command.
 
-To log in to Microsoft 365 using a certificate, you will typically create a custom Azure AD application. To use this application with the CLI for Microsoft 365, you will set the `CLIMICROSOFT365_AADAPPID` environment variable to the application's ID and the `CLIMICROSOFT365_TENANT` environment variable to the ID of the Azure AD tenant, where you created the Azure AD application.
+To log in to Microsoft 365 using a certificate, you will typically create a custom Azure AD application. To use this application with the CLI for Microsoft 365, you will set the `CLIMICROSOFT365_AADAPPID` environment variable to the application's ID and the `CLIMICROSOFT365_TENANT` environment variable to the ID of the Azure AD tenant, where you created the Azure AD application. 
 
 Managed identity in Azure Cloud Shell is the identity of the user. It is neither system- nor user-assigned and it can't be configured. To log in to Microsoft 365 using managed identity in Azure Cloud Shell, set `authType` to `identity` and don't specify the `userName` option.
 
+See the following pages for more detailed information:  
```

Comment:
Would these links be a good fit for the `More information` section that we typically use like: https://pnp.github.io/cli-microsoft365/cmd/aad/approleassignment/approleassignment-add/#more-information?

Reply:
Maybe, more information generally points to external links. These are links to three internal articles that describe some more detail the steps to take when doing the authentication. I went for the login command to find my information, if it wasn't for someone on twitter providing me with the link to the Caveats article, I wouldn't have found it.
I feel they fit in that section, but maybe as part of the text, not as a summary. Like some sort of intro that points to the first article, mentioning the caveats article somewhere in the flow of the text. It would take some more effort into rewriting this part and still keep the flow.

Reply:
Would you like to give it a try and rewrite the body to include the links in content?

Reply:
I'll put it on the list 😃. If anyone else wants to take a jab at it before I get around to it, I will not feel offended 😉

Reply:
@waldekmastykarz I made some effort to put the links in the flow of the text.

Reply:
Awesome! We'll go through your suggestions asap. Thank you 👏